lib/ecto/multi.ex

```diff
@@ -274,17 +274,17 @@ defmodule Ecto.Multi do
   The function should return either `{:ok, value}` or `{:error, value}`, and
   receives changes so far as an argument.
   """
-  @spec run(t, name, run) :: t
-  def run(multi, name, run) when is_function(run, 1) do
-    add_operation(multi, name, {:run, run})
+  @spec run(t, name, fun) :: t
```

Comment:
`run` is a public type and so we can't change that to `fun`. Because of this type I think it makes sense to use `run` as is currently being done, and not rename to `fun` below.